BDArmory/Competition/RocketChaseStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections;
+using UnityEngine;
+namespace BDArmory.Competition
+{
+    public class RocketChaseStrategy : OrchestrationStrategy
```

Comment:
Again, not strictly needed for this PR, but lays the structural ground work for a future scenario.